tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -772,6 +818,23 @@ platform_tests/test_advanced_reboot.py::test_fast_reboot_from_other_vendor:
     conditions:
       - https://github.com/sonic-net/sonic-mgmt/issues/11007
 
+#######################################
+#####   test_auto_negotiation.py  #####
+#######################################
+platform_tests/test_auto_negotiation.py:
+  skip:
+    reason: "auto negotiation test highly depends on test enviroments, file issue to track and skip for now"
+    conditions: https://github.com/sonic-net/sonic-mgmt/issues/5447
```

Comment:
This issue is closed. Will this still be skipped?

Reply:
If issue is closed, then testcase will not be skipped.